README.md

```diff
@@ -183,6 +183,13 @@ If you want to force the usage of another digest method, you should pass it as a
     ApiAuth.authentic?(signed_request, secret_key, :digest => 'sha256')
 ```
 
+For security, requests dated outside of a certain timespan are considered inauthentic.
+The default span is 15 minutes, but you can override this:
+
+```ruby
+    ApiAuth.authentic?(signed_request, secret_key, :timespan => 1.minute)
```

Comment:
How do we make it clear that "timespan" is not the entire duration as to which the request is valid, but instead the amount of time on each side of the request timestamp.